PWGHF/DataModel/HFSecondaryVertex.h

```diff
@@ -42,12 +42,18 @@ namespace hf_seltrack
 DECLARE_SOA_COLUMN(IsSelProng, isSelProng, int); //!
 DECLARE_SOA_COLUMN(DCAPrim0, dcaPrim0, float);   //!
 DECLARE_SOA_COLUMN(DCAPrim1, dcaPrim1, float);   //!
+DECLARE_SOA_COLUMN(PxProng, pxProng, float);     //!
+DECLARE_SOA_COLUMN(PyProng, pyProng, float);     //!
+DECLARE_SOA_COLUMN(PzProng, pzProng, float);     //!
```

Comment:
Is this just to speed up? Is it filled with the propagated track information?

Reply:
Hi @jgrosseo, 
yes indeed this is for speed up (see comment above) and it is before the propagation to the secondary vertex (actually before the secondary vertex reconstruction) essentially to avoid to recompute `px`, `py`, and `pz` in the preselection methods https://github.com/AliceO2Group/O2Physics/blob/b189d8a1de937d29583c7f14339964a03a255f2d/PWGHF/TableProducer/HFTrackIndexSkimsCreator.cxx#L730-L733 (which are called once for candidate, that in Pb-Pb is much larger than the number of tracks).
I also tried with expression columns, but in that case the execution time increases significantly (see above).

Reply:
OK, we follow this up offline. You can go ahead with this PR in the meanwhile.